File: db2/jruby/read/db2_read.rb
#! /usr/bin/jruby
# -*- encoding: utf-8 -*-
#
#	db2_read.rb
#
#					Jun/09/2011
#
#
# ---------------------------------------------------------------------
include Java
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
# ---------------------------------------------------------------------
puts	"*** 開始 ***"

Java::JavaClass.for_name("COM.ibm.db2.jdbc.app.DB2Driver")

url = "jdbc:db2:cities"

user = "scott"
pass = "tiger"

conn = DriverManager.getConnection(url,user,pass)

sql_read_proc(conn)

conn.close()

puts "*** 終了 ***"

# ---------------------------------------------------------------------

File: db2/jython/read/db2_read.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	db2_read.py
#					Jun/09/2011
#
# ----------------------------------------------------------------
#
import sys
from java.lang import	System
#
import java
from java.sql import DriverManager
#
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")


java.lang.Class.forName("COM.ibm.db2.jdbc.app.DB2Driver")

url = "jdbc:db2:cities"

user = "scott"
password = "tiger"

conn = DriverManager.getConnection(url,user,password)

display_proc(conn)

conn.close()

print ("*** 終了 ***")

# ---------------------------------------------------------------------
